Mlem/App/Views/Root/Tabs/Settings/ModeratorSettingsView.swift

```diff
@@ -27,11 +28,31 @@ struct ModeratorSettingsView: View {
                 Text("Separate moderator actions using...")
                     .textCase(nil)
             }
+            Section {
+                Toggle("Show All Actions in Feed", isOn: $showAllModActions)
+            } footer: {
+                // swiftlint:disable:next line_length
+                Text("When disabled, some moderator actions will be hidden from the feed and will only be visible from when viewing a post page.")
```

Comment:
*nit*

Wording can be more concise--feed behavior should be clear from toggle name, tweaked wording at the end to avoid repeating "when"

```suggestion
                Text("When disabled, some moderator actions will only be visible from the post page.")
```